rasa/nlu/extractors/duckling_entity_extractor.py

```diff
@@ -179,7 +179,7 @@ def process(self, message: Message, **kwargs: Any) -> None:
 
         if self._url() is not None:
             reference_time = self._reference_time_from_message(message)
-            matches = self._duckling_parse(message.get(TEXT), reference_time)
+            matches = self._duckling_parse(message.get(TEXT), message.get(TIME_ZONE), reference_time)
```

Comment:
This returns an `Optional[Text]` why `_duckling_parse` receives a `Text`. Either have some string as default or handle the potential `None` value